src/frontend-samples/viewer-only-2d-sample/sampleSpec.ts

```diff
@@ -3,23 +3,20 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 import { SampleSpec } from "../../Components/SampleShowcase/SampleShowcase";
-import React from "react";
-
 import { SampleIModels } from "../../Components/IModelSelector/IModelSelector";
-import ViewerOnly2dUI from ".";
+import ViewerOnly2dApp from "./ViewerOnly2dApp";
 
 export function getViewerOnly2dSpec(): SampleSpec {
   return ({
     name: "viewer-only-2d-sample",
     label: "2d",
     image: "viewer-only-2d-thumbnail.png",
     files: [
-      { name: "ViewerOnly2dSample.tsx", import: import("!!raw-loader!./index"), entry: true },
+      { name: "ViewerOnly2dApp.tsx", import: import("!!raw-loader!./ViewerOnly2dApp"), entry: true },
+      { name: "ViewerOnly2dUI.tsx", import: import("!!raw-loader!./ViewerOnly2dUI"), entry: true },
```

Comment:
I have concerns about the multiple entries when we introduce the transpiler, but that's a problem for the next PR. I think that having multiple entries will lead to having multiple bundles instead of  just one (much like how webpack handles it).